backend/pipeline/handler.go

```diff
@@ -76,7 +76,7 @@ func (p *Pipeline) HandleEvent(ctx context.Context, event *corev2.Event) error {
 
 		eventData, err := p.mutateEvent(handler, event)
 		if err != nil {
-			logger.WithError(err).Warn("error mutating event")
+			logger.WithFields(fields).WithError(err).Warn("error mutating event")
```

Comment:
Should this be at error level?

Reply:
Most likely. I wasn't sure why it was warning level to begin with. Considering it didn't even have the fields included, it probably wasn't a very useful log message!